frontend/src/bundles/studio/types/script.type.ts

```diff
@@ -1,14 +1,15 @@
 import { type ValueOf } from '~/bundles/common/types/types.js';
 
 import { type PlayIconNames } from '../enums/play-icon-names.enum.js';
+import { type Voice } from './types.js';
 
 type Script = {
     id: string;
     duration: number;
     text: string;
-    voiceName: string;
     iconName: ValueOf<typeof PlayIconNames>;
     url?: string;
+    voice?: Voice | undefined;
```

Comment:
```suggestion
    voice?: Voice;
```

Reply:
don't work without, dm you with an explanation